Procfile.dev

```diff
@@ -1,3 +1,4 @@
 web: bin/rails server -p 3000 -b 0.0.0.0
 js: yarn build --watch
 job_worker: bin/rails jobs:work
+yard: yard server --reload
```

Comment:
Checking this is intentionally included. Seems unrelated to this PR.

Reply:
Yeah it is - I'm just trying to improve the dev tooling as I go, tbh this was meant to make me want to improve the inline documentation too but ran out of energy for that

Reply:
I think my rationale here is either we are using this gem/process or we're not - having it as kind of a secret feature noone ever remembers to use is a bit weird.